pages/index.tsx

```diff
@@ -1,66 +1,12 @@
-import Head from 'next/head';
+import Layout from 'components/Layout';
 import styles from '../styles/Home.module.css';
 
 const Home: React.FC = () => {
   return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
```

Comment:
you want to keep the head, modify to make it NBJC specific? don't have to change the favicon though